addons/sale/migrations/9.0.1.0/post-migration.py

```diff
@@ -161,14 +165,14 @@ def set_so_line_computed_rest(env):
                 WHERE sol.order_id = so.id
                     AND sol.invoice_status = 'to invoice'
             ) THEN 'to invoice'
-            WHEN id = ALL(SELECT order_id FROM sale_order_line sol
-                WHERE sol.order_id = so.id
-                    AND sol.invoice_status IN ('invoiced', 'upselling'))
-                THEN 'upselling'
             WHEN id = ALL(SELECT order_id FROM sale_order_line sol
                 WHERE sol.order_id = so.id
                     AND sol.invoice_status = 'invoiced')
                 THEN 'invoiced'
+            WHEN id = ALL(SELECT order_id FROM sale_order_line sol
```

Comment:
This is sorted this way on purpose, so you are altering the logic. Take into account that `ALL` works for the select inside, so it doesn't work the same as in python.

Reply:
Now that you say that, I'm seeing both of us were wrong. Fixed now, thanks!